examples/api/houseprice.py

```diff
@@ -10,6 +11,11 @@
 lr.fit(df[["BedroomAbvGr", "YearBuilt"]], df["SalePrice"])
 
 
+@app.get("/")
+async def docs_redirect():
+    return Response("Opening the docs UI", status_code=307, headers={"location": "/docs"})
```

Comment:
@jsignell  should I change this to 301 as well . 307 is fortemporary redirect ( message that I was getting in logs)